install_backup_toolkit.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+msql_version=$1
```

Comment:
Could you also do a sanity check if only 1 parameter is passed, and handle with error message

Reply:
done please check latest version :)